packages/cli/src/utils.ts

```diff
@@ -48,3 +49,17 @@ export async function formatWithPrettier(content: string) {
     parser: "typescript",
   });
 }
+
+type BrandName = string | number | symbol;
+
+/**
+ * Create a brand type with typia
+ * I'll contribute this to typia
+ * @see https://michalzalecki.com/nominal-typing-in-typescript/#approach-4-intersection-types-and-brands
+ */
+export type Brand<TName extends BrandName> = tags.TagBase<{
+  kind: "brand type";
+  value: TName;
+  target: "any";
+  validate: never;
+}>;
```

Comment:
https://typia.io/playground/?script=JYWwDg9gTgLgBDAnmYBDANHA3g1BzAZzgF84AzKCEOAIiRVRoG4AoF+gUzgCEpUA7ACYA5VCC4BeOARhRg-PHAA+cfgFcQAIw5Rl0xFogAbViwD0AKgss4FuAGEoHVDC6o4mvkITIuAd2AYAAsfBhs7AEkAciMjOABjCH5ZYE01VwQg4CIYCFC0cLgAAQIOLiCYGDACAC4zMxBgeKDUIwAvVo54gGtgADpEkDN+KnlWgFp6eTxx+UnfAni5MBgzAGJUMDBKVGbxgBZZ5J1S+JhgJPmwDgJxgUFxz3uCcLMWDgAPSFhQrl57gA8ABVROI4J9XEIiP8hKCOAA+OBSGD4Ah9IH4bioUoArA2OC9IQ1WhPbycZj4gBurTUHGJILEHFYcFwUDwHBgxJoAkQFJZ1KMwEELjpqg4lJ0rGI8NMLES-Bk0h0lKaopkcgUcAAZDwvIIATRSlAVfEODREVJuX5bgQAMwUuVJAjGDh9IwQPAACimqD62QB6um2t1gMNytV5vhnqNJo4AEo40A

Reply:
actually it accepts plain string type so it doesn't work.
Revert it